sync_calendar.gs

```diff
@@ -103,8 +75,17 @@ function reflashCalendar(sheet) {
 
   // 新增 貼文事件 並更新 當日順序 欄位
   for(var i=0; i<reqDatas.length; i++) {
+    if(reqDatas[i].row <= 26) {
+      continue;
+    }
+    
     let tarIndex = addPostEvent(reqDatas[i].team, reqDatas[i].client, reqDatas[i].row, reqDatas[i].col, reqDatas[i].title, reqDatas[i].color);
     sheet.getRange(i+3, 5).setValue(tarIndex);
+
+    if(isTimeOut(startTime)) {
+      console.log("Time Out");
+      return;
+    }
```

Comment:
ditto

Reply:
Same explanation as this [reply](https://github.com/pycontw/pycontw-post-calendar-tool/pull/3#discussion_r1623521926).